sssd_test_framework/hosts/keycloak.py

```diff
@@ -46,6 +46,28 @@ def __init__(self, *args, **kwargs) -> None:
 
         self.adminpw = self.config.get("adminpw", "Secret123")
 
+        self.client.setdefault("id_provider", "idp")
+        self.client.setdefault("enabled", "true")
+        self.client.setdefault("idp_type", "keycloak:https://master.keycloak.test:8443/auth/admin/realms/master/")
+        self.client.setdefault("auto_private_groups", "true")
+        self.client.setdefault("use_fully_qualified_names", "true")
```

Comment:
Is this needed for all idp provider configurations?

Reply:
I would always recommend to use fully-qualified names, that's why I use them in the test as well. But I agree testing both might be even better.